Controllers/PostTodoController.cs

```diff
@@ -28,7 +29,8 @@ public PostTodoController(ITodoService todoService, IUserService userService, IM
         }
 
         [HttpPost]
-        [Authorize("Aserto")]
+        [Authorize]
+        [Check("member")]
```

Comment:
I would rename "member" to "is-resource-creator". Member doesn't really convey what this check is about.
I was confused about it until I saw the corresponding block in Startup.cs.

I think that `[Check("is-resource-creator")]` would clear things up.